src/ngraph/pass/visualize_tree.cpp

```diff
@@ -87,7 +87,9 @@ std::string pass::VisualizeTree::get_attributes(shared_ptr<Node> node)
 
     if (std::getenv("NGRAPH_VISUALIZE_TREE_OUTPUT_SHAPES") != nullptr)
     {
-        ss << " " << vector_to_string(node->get_shape());
+        //The shapes of the Outputs of a multi-output op will be printed for its corresponding `GetOutputElement`s
```

Comment:
Please keep the comment line within the 100 column limit.
Also, put a space after `//` in your comments.